test/blackbox-tests/test-cases/melange/emit-private.t

```diff
@@ -47,8 +47,8 @@ Test dependency on a private library in the same package as melange.emit
   > EOF
 
   $ OCAMLPATH=$PWD/prefix/lib/:$OCAMLPATH dune build @dist --display=short 2>&1 | grep -v melange
-          melc b/dist/a/a.js
-          melc b/dist/a/foo.js
+          melc b/dist/node_modules/a.__private__.a/a.js
+          melc b/dist/node_modules/a.__private__.a/foo.js
```

Comment:
These paths were a bit confusing for me originally because the pkg name is the same as the library. Maybe the package could be renamed to `pkg` for clarity.